snuba/snapshots/bulk_load.py

```diff
@@ -24,15 +28,17 @@ class SingleTableBulkLoader(BulkLoader):
     """
 
     def __init__(self,
-        source: BulkLoadSource,
-        dest_table: str,
-        dataset_table: str,
-    ):
+                source: BulkLoadSource,
+                dest_table: str,
+                source_table: str,
+                row_processor: Callable[[Mapping[str, Any]], Mapping[str, Any]],
```

Comment:
The callable input should probably be using `snuba.snapshot.TableRow` and `snuba.writer.TableRow` here.